source/predicates/any.ts

```diff
@@ -3,6 +3,25 @@ import {BasePredicate, testSymbol} from './base-predicate';
 import {PredicateOptions} from './predicate';
 import {Main} from '..';
 
+const generatePredicateMessage = (errors: Map<string, string[]>) => {
+	// If we have one item, return all it's errors
+	if (errors.size === 1) {
+		const array: string[] = errors.values().next().value;
+		return `Any predicate failed with the following errors:\n- ${array.join('\n- ')}`;
+	}
+
+	const errorArray = [...errors.values()];
+
+	const anyErrorWithoutOneItemOnly = errorArray.some(array => array.length !== 1);
+	// If there's any predicate with more than one error, tell users to check `validationErrors` property
+	if (anyErrorWithoutOneItemOnly) {
+		return 'Any predicate failed. Please check the `validationErrors` property for more information';
+	}
+
+	// All errors here have just one issue, report them as normal
```

Comment:
Can you end all code comments with a `.`?

Reply:
Sure sure! Let me get on my PC and I'll do them, alongside the message